src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -1,13 +1,46 @@
 package frc.team2412.robot;
 
+import static frc.team2412.robot.Hardware.HardwareConstants.BACK_LEFT_CONFIG;
+import static frc.team2412.robot.Hardware.HardwareConstants.BACK_RIGHT_CONFIG;
+import static frc.team2412.robot.Hardware.HardwareConstants.CLIMB_ANGLE_DOWN;
+import static frc.team2412.robot.Hardware.HardwareConstants.CLIMB_ANGLE_UP;
+import static frc.team2412.robot.Hardware.HardwareConstants.CLIMB_DYNAMIC;
+import static frc.team2412.robot.Hardware.HardwareConstants.CLIMB_FIXED;
+import static frc.team2412.robot.Hardware.HardwareConstants.FLYWHEEL_1;
+import static frc.team2412.robot.Hardware.HardwareConstants.FLYWHEEL_2;
+import static frc.team2412.robot.Hardware.HardwareConstants.FRONT_CAM;
+import static frc.team2412.robot.Hardware.HardwareConstants.FRONT_LEFT_CONFIG;
+import static frc.team2412.robot.Hardware.HardwareConstants.FRONT_RIGHT_CONFIG;
+import static frc.team2412.robot.Hardware.HardwareConstants.GYRO_PORT;
+import static frc.team2412.robot.Hardware.HardwareConstants.HOOD;
+import static frc.team2412.robot.Hardware.HardwareConstants.INDEX;
+import static frc.team2412.robot.Hardware.HardwareConstants.INTAKE_1;
+import static frc.team2412.robot.Hardware.HardwareConstants.INTAKE_2;
+import static frc.team2412.robot.Hardware.HardwareConstants.INTAKE_DOWN;
+import static frc.team2412.robot.Hardware.HardwareConstants.INTAKE_UP;
+import static frc.team2412.robot.Hardware.HardwareConstants.LIMELIGHT;
```

Comment:
These can all be deleted, as the following `import static ... HardwareConstants.*` line includes them

Reply:
Actually this whole block & the following frc.team2412.robot.Subsystems.SubsystemConstants entries should be able to be deleted since they duplicate existing entries below.